packages/node_modules/@node-red/registry/lib/library.js

```diff
@@ -63,7 +65,7 @@ function addNodeExamplesDir(module,path) {
     return getFlowsFromPath(path).then(function(result) {
         exampleFlows = exampleFlows||{};
         exampleFlows[module] = result;
-    });
+    }, function() { return; });
```

Comment:
From a general code style, I prefer explicit `catch` calls rather than providing second arguments to `then`. It makes the code more readable as you know what the function is there for. I had to stare at this for a while eyeing up the brackets to see what it was doing.

So this would be better written as:
```
return getFlowsFromPath(path).then(function(result) {
        exampleFlows = exampleFlows||{};
        exampleFlows[module] = result;
}).catch(function(err) { return ;});
```

Reply:
OK - noted - but no longer necessary